src/ucp/wireup/wireup_cm.c

```diff
@@ -417,6 +465,9 @@ static unsigned ucp_cm_client_uct_connect_progress(void *arg)
     static const unsigned ep_init_flags_prio[] = {
         0, UCP_EP_INIT_KA_FROM_EXIST_LANES, UCP_EP_INIT_CREATE_AM_LANE_ONLY
     };
+    static const unsigned cm_pack_log_level[]  = {
+        UCS_LOG_LEVEL_DEBUG, UCS_LOG_LEVEL_DEBUG, UCS_LOG_LEVEL_DIAG
+    };
```

Comment:
IMO it's not needed: for entries 0..N-2 the log level is DEBUG, for entry N-1 the log level is DIAG

Reply:
Logically, we can add ```if-else``` to do that. With adding this ```static const unsigned array```, it could purify/clean the code.

Reply:
Done. Keep the original ```if-else```